teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -492,6 +524,35 @@ interface VcsRoot {
     val defaultBranch: String?
 }
 
+interface BuildAgent {
+    val id: BuildAgentId
+    val name: String
+    val pool: BuildAgentPool
+
+    val connected: Boolean
+    val enabled: Boolean
+    val authorized: Boolean
+    val uptodate: Boolean
```

Comment:
What does `uptodate` mean? I don't see such think in on agents pages. Does it mean that agent is not outdated? If so I would replace this property by `outdated` property (also `outdated` is a real word there as `uptodate` is not; if we want to use `uptodate` it would be better to replace it by `upToDate`).